package.json

```diff
@@ -151,6 +151,7 @@
     "babel-plugin-lodash": "3.3.4",
     "babel-plugin-styled-components": "1.12.0",
     "babel-plugin-typescript-to-proptypes": "1.4.2",
+    "blessed": "0.1.81",
```

Comment:
just went through the library, it looks awesome. This library is last updated in 2016 but i wouldn't be too concerned since our usage is somewhat limited

Reply:
Yeah, it's pretty sweet. I'm not too worried about it being out of date, especially because it doesn't seem like something that'd change very often any way